en/advanced_config/bootloader_update_from_betaflight.md

```diff
@@ -24,7 +24,39 @@ You should now be able to install PX4 firmware on the board.
 
 #### Download Bootloader Source
 
-Download and build the [Bootloader](https://github.com/PX4/Bootloader) via:
+Flight controllers that have bootloader PX4-Autopilot `make` targets, can build the bootloader from the PX4-Autopilot source. 
```

Comment:
FYI I have reworded this to be first because there turns out to be quite a lot of FCs you can flash in this way. OK?

Reply:
This looks great. I originally wanted to try this layout since it's the only one that's worked for the FCs I have used (including matek).
I am glad you included the 'make list_config_targets'